openedx/core/djangoapps/catalog/tests/test_utils.py

```diff
@@ -23,7 +23,7 @@ def setUp(self):
         super(TestGetPrograms, self).setUp()
 
         self.user = UserFactory()
-        self.uuid = str(uuid.uuid4())
+        self.primary_uuid = str(uuid.uuid4())
```

Comment:
Looks like this might be left over from experimenting with two UUIDs? I would revert changes to this file if so.